app/lib/common/widgets/chat/convo_card.dart

```diff
@@ -218,29 +218,21 @@ class _SubtitleWidget extends ConsumerWidget {
               mainAxisSize: MainAxisSize.min,
               children: [
                 Flexible(
-                  child: Padding(
-                    padding: const EdgeInsets.symmetric(vertical: 10),
-                    child: Text(
-                      '${simplifyUserId(sender)}: ',
-                      style: Theme.of(context)
-                          .textTheme
-                          .bodySmall!
-                          .copyWith(fontWeight: FontWeight.w700),
-                      overflow: TextOverflow.ellipsis,
-                    ),
+                  child: Text(
+                    '${simplifyUserId(sender)}: ',
+                    style: Theme.of(context)
+                        .textTheme
+                        .labelMedium!
+                        .copyWith(fontWeight: FontWeight.w700),
+                    overflow: TextOverflow.ellipsis,
                   ),
                 ),
                 Flexible(
                   child: Html(
-                    padding: const EdgeInsets.symmetric(vertical: 10),
                     // ignore: unnecessary_string_interpolations
                     data: '''$body''',
                     maxLines: 1,
-                    defaultTextStyle: const TextStyle(
-                      overflow: TextOverflow.ellipsis,
```

Comment:
Wouldn't this throw UI overflow errors if it can't adjust text based on resizing or screen width?

Reply:
Resolved by https://github.com/acterglobal/a3/pull/1974/commits/6b0537a78daba6b6bac6f53a5538ee6f71bbb4dd